src/main/java/kr/codesquad/freddie/chess/board/File.java

```diff
@@ -41,6 +43,20 @@ public boolean isAddable() {
         return size() < SIZE;
     }
 
+    public void fillWith(Color color) {
+        for (; size() < SIZE; pawns.add(new Pawn(color))) ;
+    }
```

Comment:
for문의 마법사 같아요.
저라면 for문에는 로직을 넣기 보다는 while을 활용했을 것 같네요.
```suggestion
    public void fillWith(Color color) {
        while (isAddable()) {
            pawns.add(new Pawn(color));
        }
    }
```

너무 축약형만 생각하다보면, 가독성이 떨어지게됩니다.
또한, 중복도 제거해보았어요.

Reply:
동의합니다.
저는 개인적으로 for loop 를 할 때의 조건문에 활용되는 변수는, for loop block 안에 한정되는 지역변수이어야 한다고 생각합니다.

[이펙티브 자바: [아이템 57] 지역변수의 범위를 최소화하라](https://github.com/java-squid/effective-java/tree/master/chapter09/item57)

Reply:
정말 for moon의 마법사이시네요. 전 forEach를 썼어요.

Reply:
지역변수는 사용되지 않았다고 생각하는데, 어떤 부분을 말씀해주신건가요?

Reply:
지역변수가 없어서 말씀하신 것 같아요~